browser/themes/theme_properties_unittest.cc

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
++ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
++ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/themes/theme_properties.h"
+#include "chrome/browser/themes/theme_properties.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+TEST(BraveThemeTest, ObtiansBraveOverrideColors) {
+  SkColor actualColor = ThemeProperties::GetDefaultColor(ThemeProperties::COLOR_FRAME, false);
+  SkColor expectedColor = SkColorSetRGB(0x58, 0x5B, 0x5E);
```

Comment:
I assume this test will fail in official build because official uses different color for COLOR_FRAME.